api/python/cellxgene_census/pyproject.toml

```diff
@@ -68,9 +68,65 @@ exclude = ["tests*"]  # exclude packages matching these glob patterns (empty by
 [tool.setuptools_scm]
 root = "../../.."
 
-[tool.black]
+[tool.ruff]
+src = ["api/python/cellxgene_census/src"]
 line-length = 120
-target_version = ['py39']
+extend-include = ["*.ipynb"]
+target-version = "py39"
```

Comment:
Should we downgrade this to py38? Seems like a lot of the incompatible autofixes might be coming from this

Reply:
Yes, this should be py38. Existing config is a bug.

Reply:
note that the tools directory can use py311 as we only use that in-house